frontend/packages/core/src/Assets/icons/ExperimentIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
```

Comment:
Can we get Kanchan to provide this without the holes in the beaker?

Reply:
yeah I was thinking we should remove all the holes and replace them with white or off-white or whatever - the flame has a hole in the middle, the rocket ship has a hole for the window, the gem has holes as edges, and the beaker has holes as bubbles.

Reply:
I think it's ok to keep the cutouts for the various icons but I think the beaker already has lines on neck which is fine, the bubbles are just hard to see at smaller sizes